LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2016 GoDaddy Operating Company, LLC.
```

Comment:
Questionable. Shouldn't the date only be updated on major rewrites?

Reply:
It doens't need to be updated at all iirc, but it doens't hurt either.